### ISSUE TITLE
Fontbakery results for v1.006

### ISSUE THREAD

**Issue by m4rc1e** [smc/manjari#1]
We need to fix the failing tests before we can onboard the family.

## Fontbakery report

<details>
<summary><b>[2] Family checks</b></summary>
<details>
<summary>:fire: <b>FAIL:</b> Check font has a license.</summary>

* [com.google.fonts/check/028](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/028)
* :fire: **FAIL** No license file was found. Please add an OFL.txt or a LICENSE.txt file. If you are running fontbakery on a Google Fonts upstream repo, which is fine, just make sure there is a temporary license file in the same folder. [code: no-license]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have consistent PANOSE family type?</summary>

* [com.google.fonts/check/010](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/010)
* :fire: **FAIL** PANOSE family type is not the same accross this family. In order to fix this, please make sure that the panose.bFamilyType value is the same in the OS/2 table of all of this family font files.

</details>
<br>
</details>
<details>
<summary><b>[10] Manjari-Bold.ttf</b></summary>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal numbers of glyphs?</summary>

* [com.google.fonts/check/011](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/011)
* :fire: **FAIL** Bold has 780 glyphs while Regular has 865 glyphs.
* :fire: **FAIL** Thin has 857 glyphs while Regular has 865 glyphs.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal glyph names?</summary>

* [com.google.fonts/check/012](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/012)
* :fire: **FAIL** Glyphname 'uni0D78' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni005C' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni00AF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00E6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00AE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0152' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0158' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'y1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00A9' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0174' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_rh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0168' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0176' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0117' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'shm1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni012B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0124' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0177' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D4' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017C' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_m1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0120' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00DE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni0169' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0128' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni012A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FD' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni016E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0147' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00FE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_k1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni2022' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_s1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni016B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0153' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00F0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'p1th1u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_n1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B2' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 't4r3u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00AB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th3l3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00BB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00DF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0101' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni005F' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname '_th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'y1th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014D' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'z1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu2' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0125' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FC' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0121' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0116' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0100' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014C' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni0179' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00D7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0175' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017D' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0113' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B9' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00E5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0112' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0000' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni01D2' is defined on Bold, Thin but is missing on Regular.

</details>
<details>
<summary>:fire: <b>FAIL:</b> "License URL matches License text on name table?</summary>

* [com.google.fonts/check/030](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/030)
* :fire: **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache or Open Font License. For a small set of legacy families the Ubuntu Font License may be acceptable as well. [code: no-license-found]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has ttfautohint params? </summary>

* [com.google.fonts/check/has_ttfautohint_params](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/has_ttfautohint_params)
* :fire: **FAIL** Font is lacking ttfautohint params on its version strings on the name table.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Is 'gasp' table set to optimize rendering?</summary>

* [com.google.fonts/check/062](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/062)
* :fire: **FAIL** Font is missing the 'gasp' table. Try exporting the font with autohinting enabled.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has all expected currency sign characters?</summary>

* [com.google.fonts/check/070](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/070)
* :fire: **FAIL** Font lacks "EURO SIGN" character (unicode: 0x20AC)

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking OS/2 usWinAscent & usWinDescent.</summary>

* [com.google.fonts/check/040](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/040)
* :fire: **FAIL** OS/2.usWinAscent value should be equal or greater than 2076, but got 1500 instead [code: ascent]
* :fire: **FAIL** OS/2.usWinDescent value should be equal or greater than 1285, but got 900 instead [code: descent]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font enables smart dropout control in "prep" table instructions?</summary>

* [com.google.fonts/check/072](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/072)
* :fire: **FAIL** 'prep' table does not contain TrueType  instructions enabling smart dropout control. To fix, export the font with autohinting enabled, or run ttfautohint on the font, or run the  `gftools fix-nonhinting` script.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking with Microsoft Font Validator.</summary>

* [com.google.fonts/check/037](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/037)
* :fire: **FAIL** MS-FonVal: A unicode range was indicated in ulUnicodeRange, but the font has no characters in that range DETAILS: No 'Latin Extended-B' characters are present
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 8, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Intersecting contours DETAILS: Glyph index 8
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 12, Composite  Maximal deviation=16(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 19, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 23, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 24, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 25, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 26, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 27, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 28, Composite  Maximal deviation=8(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 109, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 137, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 220, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 221, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 222, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 223, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 327, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 421, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Intersecting contours DETAILS: Glyph index 460
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 648, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 708, Composite  Maximal deviation=6(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 764, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 766, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: An empty component of the composite glyph DETAILS: Glyph index 770, Component index 3
* :fire: **FAIL** MS-FonVal: The table doesn't contain strings for Mac platform
* :fire: **FAIL** MS-FonVal: The PostScript string is not present for both required platforms DETAILS: Mac Postscript string is missing, but MS Postscript string is present

</details>
<details>
<summary>:fire: <b>FAIL:</b> Name table entries should not contain line-breaks.</summary>

* [com.google.fonts/check/057](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/057)
* :fire: **FAIL** Name entry LICENSE_DESCRIPTION on platform WINDOWS contains a line-break.

</details>
<br>
</details>
<details>
<summary><b>[10] Manjari-Regular.ttf</b></summary>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal numbers of glyphs?</summary>

* [com.google.fonts/check/011](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/011)
* :fire: **FAIL** Bold has 780 glyphs while Regular has 865 glyphs.
* :fire: **FAIL** Thin has 857 glyphs while Regular has 865 glyphs.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal glyph names?</summary>

* [com.google.fonts/check/012](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/012)
* :fire: **FAIL** Glyphname 'uni0D78' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni005C' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni00AF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00E6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00AE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0152' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0158' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'y1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00A9' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0174' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_rh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0168' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0176' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0117' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'shm1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni012B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0124' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0177' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D4' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017C' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_m1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0120' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00DE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni0169' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0128' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni012A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FD' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni016E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0147' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00FE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_k1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni2022' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_s1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni016B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0153' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00F0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'p1th1u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_n1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B2' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 't4r3u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00AB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th3l3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00BB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00DF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0101' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni005F' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname '_th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'y1th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014D' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'z1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu2' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0125' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FC' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0121' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0116' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0100' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014C' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni0179' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00D7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0175' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017D' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0113' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B9' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00E5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0112' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0000' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni01D2' is defined on Bold, Thin but is missing on Regular.

</details>
<details>
<summary>:fire: <b>FAIL:</b> "License URL matches License text on name table?</summary>

* [com.google.fonts/check/030](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/030)
* :fire: **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache or Open Font License. For a small set of legacy families the Ubuntu Font License may be acceptable as well. [code: no-license-found]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has ttfautohint params? </summary>

* [com.google.fonts/check/has_ttfautohint_params](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/has_ttfautohint_params)
* :fire: **FAIL** Font is lacking ttfautohint params on its version strings on the name table.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Is 'gasp' table set to optimize rendering?</summary>

* [com.google.fonts/check/062](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/062)
* :fire: **FAIL** Font is missing the 'gasp' table. Try exporting the font with autohinting enabled.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has all expected currency sign characters?</summary>

* [com.google.fonts/check/070](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/070)
* :fire: **FAIL** Font lacks "EURO SIGN" character (unicode: 0x20AC)

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking OS/2 usWinAscent & usWinDescent.</summary>

* [com.google.fonts/check/040](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/040)
* :fire: **FAIL** OS/2.usWinAscent value should be equal or greater than 2076, but got 1500 instead [code: ascent]
* :fire: **FAIL** OS/2.usWinDescent value should be equal or greater than 1285, but got 750 instead [code: descent]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font enables smart dropout control in "prep" table instructions?</summary>

* [com.google.fonts/check/072](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/072)
* :fire: **FAIL** 'prep' table does not contain TrueType  instructions enabling smart dropout control. To fix, export the font with autohinting enabled, or run ttfautohint on the font, or run the  `gftools fix-nonhinting` script.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking with Microsoft Font Validator.</summary>

* [com.google.fonts/check/037](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/037)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 12, Composite  Maximal deviation=16(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 28, Composite  Maximal deviation=8(FU)
* :fire: **FAIL** MS-FonVal: Intersecting contours DETAILS: Glyph index 120
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 333, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 334, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 336, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 682, Composite  Maximal deviation=20(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 683, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 684, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 685, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 686, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 687, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 688, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 689, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 692, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 697, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 701, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 703, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 706, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 707, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 720, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 727, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 735, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 744, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 746, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 764, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 780, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 781, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 807, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 808, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 812, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 813, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 814, Composite  Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: The table doesn't contain strings for Mac platform
* :fire: **FAIL** MS-FonVal: The PostScript string is not present for both required platforms DETAILS: Mac Postscript string is missing, but MS Postscript string is present

</details>
<details>
<summary>:fire: <b>FAIL:</b> Name table entries should not contain line-breaks.</summary>

* [com.google.fonts/check/057](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/057)
* :fire: **FAIL** Name entry LICENSE_DESCRIPTION on platform WINDOWS contains a line-break.

</details>
<br>
</details>
<details>
<summary><b>[10] Manjari-Thin.ttf</b></summary>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal numbers of glyphs?</summary>

* [com.google.fonts/check/011](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/011)
* :fire: **FAIL** Bold has 780 glyphs while Regular has 865 glyphs.
* :fire: **FAIL** Thin has 857 glyphs while Regular has 865 glyphs.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Fonts have equal glyph names?</summary>

* [com.google.fonts/check/012](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/012)
* :fire: **FAIL** Glyphname 'uni0D78' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni005C' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname 'uni00AF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00E6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00AE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0152' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0158' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'y1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00A9' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0174' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_rh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0168' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0176' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0117' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'shm1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni012B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni015E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0124' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0177' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01D4' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017C' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_m1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni017A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0120' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00DE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni0169' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0128' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni012A' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FD' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni016E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0147' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00FE' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_k1' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni2022' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_s1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni016B' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0153' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00F0' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nh' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 't4r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'p1th1u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname '_n1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B2' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 't4r3u1' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00AB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00C5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th3l3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B3' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00BB' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00DF' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0101' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni005F' is defined on Thin but is missing on Bold, Regular.
* :fire: **FAIL** Glyphname '_th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'y1th1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00D8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014D' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'z1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1k1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname '_nhu2' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0125' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'p1th1u2' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00B6' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni01FC' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00F8' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0121' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0116' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0100' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni014C' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni0179' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'uni00D7' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'th2r3' is defined on Bold, Regular but is missing on Thin.
* :fire: **FAIL** Glyphname 'y1k1k1u1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni016F' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0175' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017D' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 's1m1' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0113' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni017E' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni00B9' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni00E5' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0112' is defined on Bold but is missing on Regular, Thin.
* :fire: **FAIL** Glyphname 'uni0000' is defined on Bold, Thin but is missing on Regular.
* :fire: **FAIL** Glyphname 'uni01D2' is defined on Bold, Thin but is missing on Regular.

</details>
<details>
<summary>:fire: <b>FAIL:</b> "License URL matches License text on name table?</summary>

* [com.google.fonts/check/030](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/030)
* :fire: **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache or Open Font License. For a small set of legacy families the Ubuntu Font License may be acceptable as well. [code: no-license-found]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has ttfautohint params? </summary>

* [com.google.fonts/check/has_ttfautohint_params](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/has_ttfautohint_params)
* :fire: **FAIL** Font is lacking ttfautohint params on its version strings on the name table.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Is 'gasp' table set to optimize rendering?</summary>

* [com.google.fonts/check/062](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/062)
* :fire: **FAIL** Font is missing the 'gasp' table. Try exporting the font with autohinting enabled.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font has all expected currency sign characters?</summary>

* [com.google.fonts/check/070](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/070)
* :fire: **FAIL** Font lacks "EURO SIGN" character (unicode: 0x20AC)

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking OS/2 usWinAscent & usWinDescent.</summary>

* [com.google.fonts/check/040](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/040)
* :fire: **FAIL** OS/2.usWinAscent value should be equal or greater than 2076, but got 1400 instead [code: ascent]
* :fire: **FAIL** OS/2.usWinDescent value should be equal or greater than 1285, but got 800 instead [code: descent]

</details>
<details>
<summary>:fire: <b>FAIL:</b> Font enables smart dropout control in "prep" table instructions?</summary>

* [com.google.fonts/check/072](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/072)
* :fire: **FAIL** 'prep' table does not contain TrueType  instructions enabling smart dropout control. To fix, export the font with autohinting enabled, or run ttfautohint on the font, or run the  `gftools fix-nonhinting` script.

</details>
<details>
<summary>:fire: <b>FAIL:</b> Checking with Microsoft Font Validator.</summary>

* [com.google.fonts/check/037](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/037)
* :fire: **FAIL** MS-FonVal: A unicode range was indicated in ulUnicodeRange, but the font has no characters in that range DETAILS: No 'Latin Extended-B' characters are present
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 12, Composite  Maximal deviation=16(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 205, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Intersecting contours DETAILS: Glyph index 651
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 685, Composite  Maximal deviation=23(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 686, Simple     Maximal deviation=1(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 689, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 690, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 702, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 703, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 707, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 709, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 711, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 712, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 725, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 728, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 736, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 740, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 745, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 749, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 779, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 781, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 782, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 790, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 791, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 807, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 810, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: Bounding box is incorrect DETAILS: Glyph index 811, Composite  Maximal deviation=2(FU)
* :fire: **FAIL** MS-FonVal: An empty component of the composite glyph DETAILS: Glyph index 847, Component index 3
* :fire: **FAIL** MS-FonVal: The table doesn't contain strings for Mac platform
* :fire: **FAIL** MS-FonVal: The PostScript string is not present for both required platforms DETAILS: Mac Postscript string is missing, but MS Postscript string is present

</details>
<details>
<summary>:fire: <b>FAIL:</b> Name table entries should not contain line-breaks.</summary>

* [com.google.fonts/check/057](https://github.com/googlefonts/fontbakery/search?q=com.google.fonts/check/057)
* :fire: **FAIL** Name entry LICENSE_DESCRIPTION on platform WINDOWS contains a line-break.

</details>
<br>
</details>

### Summary

| :broken_heart: ERROR | :fire: FAIL | :warning: WARN | :zzz: SKIP | :information_source: INFO | :bread: PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 32 | 19 | 138 | 15 | 154 |
| 0% | 9% | 5% | 39% | 4% | 43% |

**Note:** The following loglevels were omitted in this report:
* **WARN**
* **SKIP**
* **INFO**
* **PASS**

**Comment by santhoshtr** [comment]
I am working on this. Following fixes were done today:

- c33551061a303ff3110e0265cd94d8815130cf69
- ec23340e0717562ef88e991b7b9756831af3ffe2
- 323abec98a1157d7a94a1c8fd905383fbbebba63
- 0da7b005a06ee89ea2c3827f48d8e6296738c102
- a5d2a3ca4e632062d1a62790cf32e2ce1aa8144a
- 89351e37d7dab77ea29ba6c4105fd457acef6bd5
- 7d22ad1cbac31edb4f5be9cd5010e32dc78f19d4

**Comment by santhoshtr** [comment]
Report based on latest TTFs http://35.225.170.228/report/ca01c404-c05d-441c-8990-56d9fee2a5d0

@m4rc1e 

- The font is not going to be autohinted, so I guess I can ignore ttfautohint related errors. 
- The winAscend and Descend values are different for Bold, Thin, Regular as per design. Not sure that error pops up while running family test, but passes when individual fonts are tested.
- By design, certain ligatures are not vertically stacked in bold and thin for the legibility purpose. I will check if there are glyphs that not fall into this category. Otherwise, the mismatch in number of glyphs is as per design

**Comment by m4rc1e** [comment]
> The font is not going to be autohinted, so I guess I can ignore ttfautohint related errors.

Ok, I'll file an issue for FB.


> The winAscend and Descend values are different for Bold, Thin, Regular as per design. Not sure that error pops up while running family test, but passes when individual fonts are tested.

All fonts in a family should have the same winAscent and winDescent values.


> By design, certain ligatures are not vertically stacked in bold and thin for the legibility purpose. I will check if there are glyphs that not fall into this category. Otherwise, the mismatch in number of glyphs is as per design

No problem


Thanks for doing this.

**Comment by santhoshtr** [comment]
I have been actively working on this. The latest report is [http://35.225.170.228/report/dec01b30-2f9d-4382-976c-be094397b39f](here)

4 failures are present for each variant. They are:

1. com.google.fonts/check/011 Fonts have equal numbers of glyphs? Bold variant 
 * 6 ligatures were omitted from Bold variant - The stacked ligatures were horizontally distributed for better readability. This is by design.
2. com.google.fonts/check/012 Fonts have equal glyph names?
 * Same as above, Bold has 6 glyphs less. There was an issue in fontbakery related to the reporting. [I fixed that in fontbakery ](https://github.com/googlefonts/fontbakery/pull/2114)
3. com.google.fonts/check/has_ttfautohint_params Font has ttfautohint params? 
 * The font is not ttfautohinted. [I changed the behavior of the test in fontbakery](https://github.com/googlefonts/fontbakery/issues/2115). With the master version of fontbakery this test will not fail.
4. com.google.fonts/check/037 Checking with Microsoft Font Validator.
 * All three variants has 2 unresolved failures from Microsoft Font Validator. They are : 
(a) MS-FonVal: The table doesn't contain strings for Mac platform
(b) MS-FonVal: The PostScript string is not present for both required platforms DETAILS: Mac Postscript string is missing, but MS Postscript string is present. 
As per my understanding Mac platform specific [name table entries are not  important as it used to be](https://github.com/googlei18n/ufo2ft/pull/66). fontmake has no option to set these platform specific name table entries too. Correct me if I am wrong about this.

I am planning to do a release based on all these fixes done so far. 
@m4rc1e, Please let me know if you are satisified with these fixes and the above items are ignorable.

**Comment by m4rc1e** [comment]
> As per my understanding Mac platform specific name table entries are not important as it used to be. fontmake has no option to set these platform specific name table entries too. Correct me if I am wrong about this.

You are correct. I will ask to drop the Mac name checks in FB.

> I am planning to do a release based on all these fixes done so far.

Thank you for your work on this. I'm happy with your explanations. Please cut a release, I'll then do a quick QA myself.

**Comment by santhoshtr** [comment]
I just released new version: Here is the release info:

* Release: https://github.com/smc/manjari/releases/tag/Version1.700
* Zip file with ttf, otf, woff2: https://smc.org.in/downloads/fonts/manjari/manjari.zip

You may have already noticed, we use gitlab as primary repo, So canonical release tag is https://gitlab.com/smc/fonts/manjari/tags/Version1.700. But feel free to use github or gilab for this project.

**Comment by santhoshtr** [comment]
@m4rc1e ping :)

**Comment by m4rc1e** [comment]
@santhoshtr thanks man! I have a lot on this week but I'll try and get it pushed to google/fonts

Thank you for your work on this.

**Comment by m4rc1e** [comment]
@santhoshtr Apologies there's an issue which our tools should've caught.

Could we change the copyright in the fonts to `Copyright 2018 The Manjari Project Authors (https://github.com/smc/manjari)`. Could we also drop the reserved font name?

I've sent an issue to Fontbakery to catch this issue in the future, https://github.com/googlefonts/fontbakery/issues/2210

**Comment by santhoshtr** [comment]
ok, Please see 8aca7184462f13f0465c64939631881984376709

If that is fine, I will do a point release.

**Comment by santhoshtr** [comment]
I just released new version: Here is the release info:

  - Release: https://github.com/smc/manjari/releases/tag/Version1.710
  - Zip file with ttf, otf, woff2: https://smc.org.in/downloads/fonts/manjari/manjari.zip

**Comment by santhoshtr** [comment]
@m4rc1e ping. Anything else you need?

**Comment by davelab6** [comment]
@santhoshtr sorry for our delay the last ~18 months with this, we've been focused on "variable fonts" last year, and improving our onboarding/QA tools. Marc is unlikely to work on this, but I'll see if I can allocate someone else. Meanwhile, https://github.com/googlefonts/gf-docs/tree/master/Spec is new and fresh and may have some tips for you :)

**Comment by santhoshtr** [comment]
This ticket is no longer relevant as Manjari was added to Google fonts about an year ago: https://github.com/google/fonts/pull/1843
